### DISCUSSION TITLE
به بخش گفتگو خوش اومدید

### DISCUSSION THREAD

**Discussion by rzashakeri** [barnamenevisi/free-resources#2] [Announcements]
## 👋 خوش اومدید!

این بخش برای این ساخته شده اگر از کتاب‌ها،‌دوره‌ها، کانال‌ها و ... استفاده کردید نظرتون رو دربارش بگید تا دیگران بتونن با آگاهی بیشتر انتخاب کنن و یا اگر سوالی در رابطه با این ریپازیتوری بود بپرسید

**Comment by Amirhsv** [comment]
روی دوره ها میزنم چیزی باز نمیشه
